README_TTN.md

```diff
@@ -103,6 +107,134 @@ Details are available [here](https://blog.squix.org/2017/07/seeeduino-lora-gps-g
 Compiling the OTAA sketch should make the Seeeduino connect to the Gateway and
 send messages to the TTN.
 
+## Receive data from TTN into IoT-LoRaWan Agent
+
+In order to receive data from TTN the IoT-Lorawan Agent should
+subscribe to the applications on the TTN platform.
+
+The code below is the comand, already configured, that we use in order to connect.
+
+Two keys, one for each application are generated in order to
+connect and used as password.
+Remember to remove the security code.
+
+```bash
+curl --location --request POST 'localhost:4042/iot/services' \
+--header 'fiware-service: openiot' \
+--header 'fiware-servicePath: /' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "services": [
+    {
+      "entity_type": "waste",
+      "apikey": "",
+      "resource": "8CF957200005727C",
+      "attributes": [
+        {
+          "object_id": "temperature_1",
+          "name": "temperature",
+          "type": "Number"
+        },
+        {
+          "object_id": "relative_humidity_1",
+          "name": "relative_humidity",
+          "type": "Number"
+          },
+          {
+            "object_id": "digital_in_1",
+            "name": "flame_detected",
+            "type": "Number"
+            },
+            {
+              "object_id": "digital_in_2",
+              "name": "distance",
+              "type": "Number"
+            },
+            {
+              "object_id": "accelerometer_1",
+              "name": "accelerometerXYZ",
+              "type": "StructuredValue"
+            },
+            {
+              "object_id": "accelerometer_2",
+              "name": "gAccellerationXYZ",
+              "type": "StructuredValue"
+            }
+          ],
+          "internal_attributes": {
+            "lorawan": {
+              "application_server": {
+                "host": "eu1.cloud.thethings.network",
+                "username": "oc-diorama-001@ttn",
+                "password": "NNSXS.KQRRO4R7BXMHELHQZOMELYUJ2GQUU4S5EI2I6QY.P7FJVWQRODOLCOHOAC2GQMEDO65LINW6YAOBKL6YMUU2K5C4R5MAC0MM3NT01T4L1ANO",
+                "provider": "TTN"
+                },
+                "app_eui": "8CF957200005727C",
+                "application_id": "oc-diorama-001@ttn",
+                "application_key": "51F58CAC3F5735E3D1F88DD3EADBE9C6",
+                "data_model": "application_server"
+            }
+          }
+    },
+    {
+      "entity_type": "environment",
```

Comment:
see the old code:
https://gitlab.com/orchestracities/iot/-/blob/master/diorama/src/environment/create_device.sh